priv/templates/ja_serializer.gen.phoenix_api/controller.ex

```diff
@@ -0,0 +1,60 @@
+defmodule <%= module %>Controller do
+  use <%= base %>.Web, :controller
+
+  alias <%= module %>
+  alias JaSerializer.Params
+
+  plug :scrub_params, "meta" when action in [:create, :update]
+  plug :scrub_params, "data" when action in [:create, :update]
```

Comment:
@alanpeabody Do you think this is necessary? With it, you get an error like `expected key "data" to be present in params, please send the expected key or adapt your scrub_params/2 call`, and without it, you'll get an error that there are no matching functions.

Maybe not necessary, but I think it provides a more friendly error message.